elements/lisk-p2p/test/integration/p2p.ts

```diff
@@ -481,17 +470,15 @@ describe('Integration tests for P2P library', () => {
 
 			it('should not contain itself in any of its peer list', async () => {
 				for (let p2p of p2pNodeList) {
-					const {
-						triedPeers,
-						connectedPeers,
-						newPeers,
-					} = p2p.getNetworkStatus();
+					const newPeers = p2p['_peerBook'].newPeers;
```

Comment:
We can simply use `getAllPeers` method of `PeerBook` in the end we can remove the use of new tried peers and modify the related tests